### DISCUSSION TITLE
RSS feed for changes, updates and releases?

### DISCUSSION THREAD

**Discussion by NightHawkATL** [searxng/searxng#3148] [Q&A] [answered]
I use FreshRSS to monitor updates and see if there are changes to containers that I need to be aware of. I have yet to find an RSS feed or even a release page for SearxNG and I see sometimes that there are multiple updates when someone is working on it. Yesterday, it did an update and the only way I could figure out what was going on after work after getting alerts of it going up and down all day was to look at the logs to see that there were new settings files that I had to rename. It would be nice to have an update feed that myself and others can follow to see what updates are made and if there are breaking changes.

**Comment by virtadpt** [comment, marked as answer]
https://github.com/searxng/searxng/commits/master.atom

**Reply by NightHawkATL** [reply]
Thank you!
